src/converter/one/convert/FlameGraph.java

```diff
@@ -67,7 +67,7 @@ public void parseHtml(Reader in) throws IOException {
         Frame[] levels = new Frame[128];
         int level = 0;
         long total = 0;
-        boolean needRebuild = args.reverse || args.include != null || args.exclude != null;
+        boolean needRebuild = args.reverse || args.inverted || args.include != null || args.exclude != null;
```

Comment:
`inverted` affects display only - there is no need to rebuild flame graph structure if this option changes.